web/src/components/tasks/HousingRegistrationStatusSummary.tsx

```diff
@@ -0,0 +1,170 @@
+import { HorizontalLine } from "@/components/HorizontalLine";
+import { UserDataErrorAlert } from "@/components/UserDataErrorAlert";
+import { UnStyledButton } from "@/components/njwds-extended/UnStyledButton";
+import { Icon } from "@/components/njwds/Icon";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { Task } from "@/lib/types/types";
+import { toProperCase } from "@businessnjgovnavigator/shared";
+import { parseDate } from "@businessnjgovnavigator/shared/dateHelpers";
+import {
+  HousingAddress,
+  HousingRegistrationRequestLookupResponse,
+} from "@businessnjgovnavigator/shared/housing";
+import { Box } from "@mui/material";
+import { ReactElement } from "react";
+
+interface Props {
+  task: Task;
+  summary: HousingRegistrationRequestLookupResponse;
+  address: HousingAddress | undefined;
+  onEdit: () => void;
+}
+
+type CardDisplayDetails = {
+  headerColor: string;
+  bodyColor: string;
+  icon: string;
+  iconTextColor: string;
+};
+
+export const HousingRegistrationStatusSummary = (props: Props): ReactElement => {
+  const { Config } = useConfig();
+
+  const { business } = useUserData();
+
+  const getDetailsForRegistrationCard = (status: string): CardDisplayDetails => {
+    switch (status) {
+      case "Approved":
+        return {
+          headerColor: "bg-success-darker",
+          bodyColor: "bg-success-lighter",
+          icon: "check_circle",
+          iconTextColor: "text-success-darker",
+        };
+      case "Rejected":
+      case "Returned":
+      case "Incomplete":
+      case "Cancelled":
+        return {
+          headerColor: "bg-error",
+          bodyColor: "bg-error-extra-light",
+          icon: "cancel",
+          iconTextColor: "text-error-darker",
+        };
+      default:
+        return {
+          headerColor: "bg-info-dark",
+          bodyColor: "bg-info-extra-light",
+          icon: "",
+          iconTextColor: "",
+        };
+    }
+  };
+
+  const getInformationalMessageText = (status: string): string => {
+    switch (status) {
+      case "In Review":
+        return Config.housingRegistrationSearchTask.informationalInReview;
+      case "Incomplete":
+        return Config.housingRegistrationSearchTask.informationalIncomplete;
+      case "Rejected":
+        return Config.housingRegistrationSearchTask.informationalRejected;
+      case "Returned":
+        return Config.housingRegistrationSearchTask.informationalReturned;
+      default:
+        return "";
+    }
+  };
+
+  const housingRegistrationStatusCard = (date: string, status: string, index: number): ReactElement => {
```

Comment:
Opportunity for a refactor since the cards are similar between Housing and Elevator - added a ticket to the muni backlog to do this

Reply:
cool